Lib/pathlib.py

```diff
@@ -1284,6 +1287,14 @@ def rmdir(self):
             self._raise_closed()
         self._accessor.rmdir(self)
 
+    def rmtree(self, ignore_errors=False, onerror=None):
+        """
+        Remove this directory tree.
+        """
+        if self._closed:
+            self._raise_closed()
```

Comment:
There is not test for this case

Reply:
Happy to add a test case, however I noticed [other similar methods](https://github.com/asmith26/cpython/blob/fix-issue-33498/Lib/test/test_pathlib.py#L1632-L1644) also did not test this.